### PR TITLE
ROLLUP-437: complete withdrawals configuration

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- As an operator, I want to have the possibility to configure the server to allow or not the complete withdrawal. For security reasons, such configuration should NOT be allowed in Mainnet.

- Complete withdrawal: when enabled, it allows the users to move funds from their L2 accounts to their L1 accounts directly. When disabled, if the user withdraws funds from L2, the funds are stored in the L1 contract; the users need to get the pending balances to move funds from the L1 contract to their L1 addresses.

## Why

- UX

## Refs

- [ROLLUP-437](https://rsklabs.atlassian.net/browse/ROLLUP-437)

### CODE REVIEW

File: core/tests/testkit/src/eth_account.rs [hunk 1]
```diff
@@ -407,7 +407,7 @@ impl RootstockAccount {
     ) -> Result<ETHExecResult, anyhow::Error> {
         let data = self.main_contract_eth_client.encode_tx_data(
             "executeBlocks",
-            execute_operation.get_eth_tx_args().as_slice(),
+            execute_operation.get_eth_tx_args(false).as_slice(),
```
Comment (franciscotobar) [inline core/tests/testkit/src/eth_account.rs:410]: this class is used in the testkit, and it does not affect if it's true or false.

**Review by antomor — CHANGES_REQUESTED** [review]
@franciscotobar thank you for the changes 🙏🏼 
It looks like everything is working as expected apart from the [config check](https://github.com/rsksmart/rif-rollup/pull/104/files#diff-8728f16887d26920dba138292eb716ba2727cacaec2b41a8f4d2f525ecd60dd0R25). There, we check for the [chain_eth_network](https://github.com/rsksmart/rif-rollup/blob/043d3ae5cdc6bcfe5c43dd32295e21b1185cdaa0/etc/env/mainnet/chain.toml#L20), but that value is supposed to be `localhost` for now. So we should either change the value in the mainnet configuration or check for the [chain_id](https://github.com/rsksmart/rif-rollup/blob/043d3ae5cdc6bcfe5c43dd32295e21b1185cdaa0/etc/env/mainnet/eth_client.toml#L2).

Furthermore, (not related to this PR) I've realised that the [chain_id configured in api.toml](https://github.com/rsksmart/rif-rollup/blob/043d3ae5cdc6bcfe5c43dd32295e21b1185cdaa0/etc/env/mainnet/api.toml#L46C1-L46C1) is under the wrong section `[api.rest]`, because it's supposed to be `[api.web3]`.

**Comment by franciscotobar** [comment]
> @franciscotobar thank you for the changes 🙏🏼 It looks like everything is working as expected apart from the [config check](https://github.com/rsksmart/rif-rollup/pull/104/files#diff-8728f16887d26920dba138292eb716ba2727cacaec2b41a8f4d2f525ecd60dd0R25). There, we check for the [chain_eth_network](https://github.com/rsksmart/rif-rollup/blob/043d3ae5cdc6bcfe5c43dd32295e21b1185cdaa0/etc/env/mainnet/chain.toml#L20), but that value is supposed to be `localhost` for now. So we should either change the value in the mainnet configuration or check for the [chain_id](https://github.com/rsksmart/rif-rollup/blob/043d3ae5cdc6bcfe5c43dd32295e21b1185cdaa0/etc/env/mainnet/eth_client.toml#L2).
> 
> Furthermore, (not related to this PR) I've realised that the [chain_id configured in api.toml](https://github.com/rsksmart/rif-rollup/blob/043d3ae5cdc6bcfe5c43dd32295e21b1185cdaa0/etc/env/mainnet/api.toml#L46C1-L46C1) is under the wrong section `[api.rest]`, because it's supposed to be `[api.web3]`.

In testnet and qa, I have the testnet value, but I'm concerned about the comment https://github.com/rsksmart/rif-rollup/blob/ROLLUP-437/configurable-withdrawal/etc/env/mainnet/chain.toml#L19.

Maybe using the chain_id, will be better, but we will be only taken into account, RSK.

**Comment by antomor** [comment]
>Maybe using the chain_id, will be better, but we will be only taken into account, RSK.
Yes, I thought the same indeed

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**